### DISCUSSION TITLE
MariaDB Support

### DISCUSSION THREAD

**Discussion by robertsilen** [PowerDNS/pdns#14538] [Q&A] [answered]
Hello, what is the status of MariaDB support and compatibility in PowerDNS? 

I see that PowerDNS has a MySQL module (https://github.com/PowerDNS/pdns/tree/master/modules), but neither the
module nor the docs (https://doc.powerdns.com/authoritative/backends/generic-mysql.html) mention anything about MariaDB support. However in Debian the package (https://packages.debian.org/unstable/pdns-backend-mysql) depends on the default-mysql-server which in turn has been pointing to MariaDB in Debian since 2016.

Thank you for any comments on this.

**Comment by rgacogne** [comment]
Hi,

PowerDNS Authoritative does not use any feature of MySQL that is not implemented in MariaDB, and we indeed use MariaDB as the MySQL implementation on Debian and derivatives, so MariaDB is supported and know to work well.

**Reply by robertsilen** [reply]
@rgacogne Thank you for your reply and confirming MariaDB compatibility. Would it be possible to add a mention of MariaDB to the documentation at https://doc.powerdns.com/authoritative/backends/generic-mysql.html ? I can suggest a change with a PR myself to https://github.com/PowerDNS/pdns/blob/master/docs/backends/generic-mysql.rst if that's reasonable?

**Reply by rgacogne** [reply]
That sounds like a good idea indeed. A PR would be very much appreciated, thanks!

**Reply by robertsilen** [reply, marked as answer]
@rgacogne Thanks, a PR is waiting now :) https://github.com/PowerDNS/pdns/pull/14541
